apps/docs/src/app/core/component-docs/fixed-card-layout/fixed-card-layout-docs-header/fixed-card-layout-docs-header.component.html

```diff
@@ -1,13 +1,14 @@
 <header>Fixed Card Layout</header>
 <description>
     The fixed card layout supports automatic, easy and fast card design. The cards are ordered in responsive and
-    collapsible columns.The cards are arranged as a “Z” flow: from left to right, starting with the first card on the
-    top left of the page.
+    collapsible columns. The cards are arranged into a “Z” flow: from left to right, starting with the first card on the
+    top left of the page. The maximum number of columns is 10.
```

Comment:
maybe I'm missing something, but I don't see where it's handled in code. This is just relying on css, right?

Reply:
Right. I could find information only about 1-3 columns in wiki, so 10 it's more than enough i think.